ReportSimulationOutput/measure.rb

```diff
@@ -304,9 +309,23 @@ def energyPlusOutputRequests(runner, user_arguments)
       end
     end
     if has_electricity_production
-      result << OpenStudio::IdfObject.load('Output:Meter,ElectricityProduced:Facility,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Output:Meter,Photovoltaic:ElectricityProduced,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Output:Meter,Cogeneration:ElectricityProduced,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Output:Meter,Photovoltaic:Inverter,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Meter:Custom,Photovoltaic:Inverter,electricity,*,Inverter Conversion Loss Decrement Energy;').get
       if include_timeseries_fuel_consumptions
-        result << OpenStudio::IdfObject.load("Output:Meter,ElectricityProduced:Facility,#{timeseries_frequency};").get
+        result << OpenStudio::IdfObject.load("Output:Meter,Photovoltaic:ElectricityProduced,#{timeseries_frequency};").get
+        result << OpenStudio::IdfObject.load("Output:Meter,Cogeneration:ElectricityProduced,#{timeseries_frequency};").get
+        result << OpenStudio::IdfObject.load("Output:Meter,Photovoltaic:Inverter,#{timeseries_frequency};").get
+      end
+    end
+    if has_electricity_storage
+      result << OpenStudio::IdfObject.load('Output:Meter,ElectricStorage:ElectricityProduced,runperiod;').get
+      result << OpenStudio::IdfObject.load('Output:Meter,ElectricStorage:Converter,runperiod;').get
+      result << OpenStudio::IdfObject.load('Meter:Custom,ElectricStorage:Converter,electricity,*,Converter Electricity Loss Decrement Energy;').get
```

Comment:
I don't see a single reference to "Converter Electricity Loss Decrement Energy" elsewhere in the code. What is this doing?

Reply:
I think it was relevant when the converter efficiency was less than one.